src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public class ColorSupplier {
+    private final String [] colors = {"RED", "ORANGE", "GREEN", "GRAY", "YELLOW"};
```

Comment:
let's make it a constant
https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names
```suggestion
    private static final String [] COLORS = {"RED", "ORANGE", "GREEN", "GRAY", "YELLOW"};
```